### ISSUE TITLE
HTML inside component block is escaped

### ISSUE THREAD

**Issue by scottcorgan** [desiringgod/components-rails#24]
It looks like the content in a block for a component is getting escaped and not allowing us to nest HTML.

This code

<img width="303" alt="screen shot 2017-09-13 at 8 48 30 am" src="https://user-images.githubusercontent.com/974723/30378243-8ab234ac-9860-11e7-8f41-52276ecf695f.png">

produces this HTML

<img width="278" alt="screen shot 2017-09-13 at 8 48 48 am" src="https://user-images.githubusercontent.com/974723/30378252-98f04c5c-9860-11e7-97d6-5a1c07e23c69.png">

and looks like this when it is rendered

<img width="289" alt="screen shot 2017-09-13 at 8 48 21 am" src="https://user-images.githubusercontent.com/974723/30378247-930cb9ce-9860-11e7-9b60-ab7f9592f314.png">

**Note:** The HTML produces sets the `span` stuff as text content rather than inner html which is just escaping the `<` and `>` so the browser ignores those. I assume it's a `html_safe` method thing on the Ruby side or something like that.

**Comment by joeosburn-desiringgod** [comment]
@benhutton Thoughts on this? I'm having trouble reproducing this in a spec.

**Comment by benhutton** [comment]
@scottcorgan @joeosburn-desiringgod if we're seeing this in the app based on the old code, it seems a waste to try to reproduce in the gem based on the new code until we verify that it still is in the app with the new code.

TLDR: maybe you fixed it already?

**Comment by joeosburn-desiringgod** [comment]
@scottcorgan Can you verify this is happening with the latest version of the gem?

**Comment by scottcorgan** [comment]
@joeosburn-desiringgod looks to be working great with a quick test. Thanks so much!
